react-viewer/src/TestClient.ts

```diff
@@ -1,64 +1,69 @@
-import { AuthStatus, BeEvent, BentleyError, ClientRequestContext } from "@bentley/bentleyjs-core";
+import {
+  AuthStatus,
+  BeEvent,
+  BentleyError,
+  ClientRequestContext,
+} from "@bentley/bentleyjs-core";
 import { AccessToken } from "@bentley/itwin-client";
 import { FrontendAuthorizationClient } from "@bentley/frontend-authorization-client";
 
 /** This client defines the interfaces  */
 export class NoSignInIAuthClient implements FrontendAuthorizationClient {
-  public readonly onUserStateChanged: BeEvent<(token: AccessToken | undefined) => void>;
+  public readonly onUserStateChanged: BeEvent<
+    (token: AccessToken | undefined) => void
+  >;
   protected _accessToken?: AccessToken;
 
   private static _oidcClient: FrontendAuthorizationClient;
 
   public static get oidcClient(): FrontendAuthorizationClient {
-    if (undefined !== this._oidcClient)
+    if (this._oidcClient) {
       return this._oidcClient;
+    }
     this._oidcClient = new NoSignInIAuthClient();
     return this._oidcClient;
   }
 
   constructor() {
     this.onUserStateChanged = new BeEvent();
-    this.generateTokenString();
+    this.generateToken();
   }
 
   public async signIn(requestContext?: ClientRequestContext): Promise<void> {
-    if (requestContext) {
-      requestContext.enter();
-    }
-    await this.getAccessToken();
+    requestContext?.enter();
+    this.generateToken();
```

Comment:
This should be awaited or returned

Reply:
nvm, I was thinking `generateToken` was an async function like getAccessToken... Ignore this